_tkinter module needs tk module along with tcl. tk is not yet integrated
in yocto so we skip the check for this module.
Avoid a warning by not adding this module to missing variable.

Upstream-Status: Inappropriate [distribution]

Also simply disable the tk module since its not in DEPENDS.

2018-08: Rebased for 3.7.0
  -- Jens

Signed-off-by: Andrei Gherzan <andrei@gherzan.ro>

diff --git a/setup.py b/setup.py
index 37c5dd58a6..0321245761 100644
--- a/setup.py
+++ b/setup.py
@@ -1608,10 +1608,12 @@ class PyBuildExt(build_ext):
         self.extensions.extend(exts)
 
         # Call the method for detecting whether _tkinter can be compiled
-        self.detect_tkinter(inc_dirs, lib_dirs)
+        # self.detect_tkinter(inc_dirs, lib_dirs)
 
-        if '_tkinter' not in [e.name for e in self.extensions]:
-            missing.append('_tkinter')
+        # tkinter module will not be avalaible as yocto
+        # doesn't have tk integrated (yet)
+        #if '_tkinter' not in [e.name for e in self.extensions]:
+        #    missing.append('_tkinter')
 
         # Build the _uuid module if possible
         uuid_incs = find_file("uuid.h", inc_dirs, ["/usr/include/uuid"])
